scripts/sleepstudy_borrowingstrength_RStudio.Rmd

```diff
@@ -1,134 +0,0 @@
----
```

Comment:
why is this file being deleted?

Reply:
It was not deleted only renamed _lme4.Rmd